db/migrate/20151127162510_downcase_email_and_remove_duplicate_entries.rb

```diff
@@ -0,0 +1,15 @@
+class DowncaseEmailAndRemoveDuplicateEntries < ActiveRecord::Migration
+  def change
+  	members = Member.all
+  	emails = []
+  	members.each do |member|
+  		member.email.downcase!
+  		if emails.include?(member.email)
+  			member.destroy
+  		else
+  			emails<<member.email
+  			member.save!
+  		end
+  	end
```

Comment:
How about this? Checking if the email is already present in an array and deleting if its true